src/ui/UserListItem/index.tsx

```diff
@@ -58,44 +56,53 @@ export default function UserListItem({
   onClick,
   onUserAvatarClick,
   renderListItemMenu,
+  size = 'normal',
 }: UserListItemProps): ReactElement {
   const operator = isOperator ?? (user as Member)?.role === 'operator';
   const uniqueKey = user.userId;
-  const actionRef = React.useRef(null);
-  const parentRef = React.useRef(null);
-  const avatarRef = React.useRef(null);
+  const actionRef = useRef(null);
+  const parentRef = useRef(null);
+  const avatarRef = useRef(null);
   const { disableUserProfile, renderUserProfile } = useContext(UserProfileContext);
   const { stringSet } = useLocalization();
   const { config } = useSendbirdStateContext();
   const currentUser = config.userId;
 
+  const itemClassName = size === 'small' ? 'sendbird-user-list-item--small' : 'sendbird-user-list-item';
+  const avatarClassName = size === 'small' ? 'sendbird-user-list-item--small__avatar' : 'sendbird-user-list-item__avatar';
+  const titleClassName = size === 'small' ? 'sendbird-user-list-item--small__title' : 'sendbird-user-list-item__title';
+  const subtitleClassName = size === 'small' ? 'sendbird-user-list-item--small__subtitle' : 'sendbird-user-list-item__subtitle';
+  const checkboxClassName = size === 'small' ? 'sendbird-user-list-item--small__checkbox' : 'sendbird-user-list-item__checkbox';
+  const actionClassName = size === 'small' ? 'sendbird-user-list-item--small__action' : 'sendbird-user-list-item__action';
+  const operatorClassName = size === 'small' ? 'sendbird-user-list-item--small__operator' : 'sendbird-user-list-item__operator';
```

Comment:
```
const getClassName = (modifier = '') => size === 'small' ? `sendbird-user-list-item--small${modifier}` : `sendbird-user-list-item${modifier}`;
```
We can do like this too.
```
className={getClassName('__avatar')}
```

Reply:
Am... actually I've stopped shortening class names like this. I used to do it before, but it made debugging through the inspector a less pleasant experience.